sympy/combinatorics/fp_groups.py

```diff
@@ -331,7 +331,7 @@ def most_frequent_generator(self):
         return gens[freqs.index(max(freqs))]
 
     def random(self):
-        import random
+        from sympy.core.compatibility from sympy.core import local_random as random
```

Comment:
This line is invalid syntax and is causing the tests to fail

Reply:
Fixed